components/WeeklyPickButton/WeeklyPickButton.test.tsx

```diff
@@ -3,19 +3,93 @@ import { render, screen } from '@testing-library/react';
 import { WeeklyPickButton } from './WeeklyPickButton';
 import { RadioGroup } from '../RadioGroup/RadioGroup';
 
+// test one method
+// provide some specific arguments to that method
+// test that the result is as expected
+
+const weeklyPickData = {
+  team: 'Ravens',
+  src: '/path/to/ravens.svg',
+  height: '48',
+  width: '48',
+  alt: 'Ravens',
+};
+
 describe('WeeklyPickButton', () => {
-  it('renders correctly', () => {
+  it('RadioGroup has the correct classes', () => {
+    render(<RadioGroup />);
+
+    const radioGroup = screen.getByRole('radiogroup');
+    expect(radioGroup).toHaveClass('grid w-full grid-cols-2 gap-4');
+  });
+
+  it('radio button has the correct classes', () => {
     render(
       <RadioGroup>
-        <WeeklyPickButton team="Test Team" src="/path/to/image.svg"/>
+        <WeeklyPickButton team={weeklyPickData.team} src={weeklyPickData.src} />
       </RadioGroup>,
     );
 
-    const image = screen.getByRole('img', { name: /Test Team/i });
-    expect(image).toBeInTheDocument();
-    expect(image).toHaveAttribute('src', '/path/to/image.svg');
+    const radioButton = screen.getByRole('radio');
+    expect(radioButton).toHaveClass('peer sr-only');
+  });
+
+  it('image should have a height of 48', () => {
+    render(
+      <RadioGroup>
+        <WeeklyPickButton team={weeklyPickData.team} src={weeklyPickData.src} />
+      </RadioGroup>,
+    );
+
+    const image = screen.getByRole('img', { name: /Ravens/i });
+    expect(image).toHaveAttribute('height', weeklyPickData.height);
+  });
+
+  it('image should have a width of 48', () => {
+    render(
+      <RadioGroup>
+        <WeeklyPickButton team={weeklyPickData.team} src={weeklyPickData.src} />
+      </RadioGroup>,
+    );
+
+    const image = screen.getByRole('img', { name: /Ravens/i });
+    expect(image).toHaveAttribute('width', weeklyPickData.width);
+  });
 
-    const teamName = screen.getByText(/Test Team/i);
+  it('image should have path name in src', () => {
```

Comment:
All of these tests should be combined. There's no need to create a separate test for each prop when they're already in the first render.